pkg/consensus/consensus.go

```diff
@@ -119,7 +119,7 @@ func NewService(cfg Config) (Service, error) {
 		dbft.WithLogger(srv.log),
 		dbft.WithSecondsPerBlock(cfg.TimePerBlock),
 		dbft.WithGetKeyPair(srv.getKeyPair),
-		dbft.WithTxPerBlock(10000),
+		dbft.WithTxPerBlock(srv.Chain.MaxTransactionsPerBlock()),
```

Comment:
It doesn't make much sense as the value of `srv.Chain.MaxTransactionsPerBlock()` may change in any new block while dbft assumes this constant to be used for the whole dbft lifetime. And the only place this value is being used is for `GetVerified` which is `getVerifiedTx` which doesn't use this `count` in any way (and it shouldn't as it applies it's own policy).

Reply:
Removed `dbft.WithTxPerBlock` at all (from `dbft`), as we really don't need `TxPerBlock` anymore, is it OK? If so, I'll create a separate PR in `dbft`.

Reply:
Well, given that `dbft` itself doesn't use this limit in any way, it may probably be a good choice. Any kind of limiting/policying should be implemented by the client of this library. @fyrchik?

Reply:
`TxPerBlock` is only passed in `GetVerified` (which is also a hook). It (`TxPerBlock`) can be safely removed both from here and lib. If verification is needed, we have `VerifyPrepareRequest`.